style.css

```diff
@@ -1,13 +1,30 @@
-body {
-    width: 100%;
-    height: 85vh;
-}
-
-.headerContainer {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: center;
-    border-bottom: solid 4px black;
-    font-size: 40px;
+table > thead {
+    border-bottom: 1px solid black;
+}
+header {
+    display: inline-block;
+}
+.table {
+    width: 65%;
+    height: 200px;
+    margin-left: 5%;
+    margin-right: 15%;
```

Comment:
What is the purpose of the margins applied here? It might be okay, but a 15% margin is a considerable amount 🤔